sdk-estimate-gas/gas.js

```diff
@@ -6,15 +6,29 @@ const ethers = require("ethers")
 const optimismSDK = require("@eth-optimism/sdk")
 const fs = require("fs")
 require('dotenv').config()
+const yargs = require("yargs")
+const { boolean } = require("yargs")
+
+
+const argv = yargs
+  .option('network', {
+    choices: ["mainnet", "goerli", "bedrock-alpha"],
+    description: 'Optimistm network to use'
+  }).
+  option('verify', {
+    type: boolean,
+    description: 'Run the transaction, compare to the estimate'
+  })
+  .help()
+  .alias('help', 'h').argv;
 
-const greeterJSON = JSON.parse(fs.readFileSync("Greeter.json"))
 
-const greeterAddrs = {
-  // Optimism
-  "10":  "0x5825fA9cD0986F52A8Dda506564E99d24a8684D1",
+const greeterJSON = JSON.parse(fs.readFileSync("Greeter.json")) 
 
-  // Optimism Goerli
-  "420": "0x106941459A8768f5A92b770e280555FAF817576f"
+const greeterAddrs = {
+  "mainnet":  "0x5825fA9cD0986F52A8Dda506564E99d24a8684D1",
+  "goerli": "0x106941459A8768f5A92b770e280555FAF817576f",
```

Comment:
We should specify here if we mean optimism mainnet or eth mainnet, the same thing for goerli (so maybe optimism-goerli or something.